### ISSUE TITLE
Review annotations to 'regulation of protein x-merization activity' and children

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/go-annotation#2684]
Hi,

There are a few more annotations to review for the x-merization activity terms.  This ticket addresses the 'regulation of x-merization activity' terms which will be obsoleted.

Annotations to be reviewed are organized according to evidence code below and are in this Google spreadsheet:

https://docs.google.com/spreadsheets/d/18KdAXo95Ms7aYDbETTzwmHiaPm3cH6k8o-6uvEXYFlw

Further information about review of these annotations can be found in this Google doc:

https://docs.google.com/document/d/1JxzNwl1fr5Fj-IJy_Fo1lLfAlAQB8IxPDwwYKxIXBZo/edit#heading=h.oxlxrmsdkphr

Any questions or edge cases for annotation, please add a comment to the ticket and we'll discuss.

**Please also remember to mark the spreadsheet when you're done, so we can calculate accurate metrics for review efforts for the grant.**

If you have any trouble accessing the spreadsheet, please send a request for edit access.

Thank you!

**Comment by vanaukenk** [comment]
**IDA**

Annotations to regulation of protein x-merization activity and children with evidence code ECO:0000314 (IDA)

Issue:  Need manual review to possibly change to 'regulation of protein complex assembly' or a child term; alternatively, can be removed.

**UniProt** - 6
**BHF-UCL** - 3
**HGNC** - 3
**TAIR** - 3
**ParkinsonsUK-UCL** - 2
**ARUK-UCL** - 1
**dictyBase** - 1
**SGD** - 1

**Comment by vanaukenk** [comment]
**IMP**

Annotations to regulation of protein x-merization activity and children with evidence code ECO:0000315 (IMP)

Issue:  Need manual review to possibly change to 'regulation of protein complex assembly' or a child term; alternatively, can be removed.

**UniProt** - 8
**BHF-UCL** - 1
**ARUK-UCL** - 1
**dictyBase** - 1

**Comment by vanaukenk** [comment]
IPI

Annotations to regulation of protein x-merization activity and children with evidence code ECO:0000353 (IPI)

Issue: Need manual review to possibly change to 'regulation of protein complex assembly' or a child term; alternatively, can be removed.

**UniProt** - 2
**HGNC** - 1

**Comment by vanaukenk** [comment]
**TAS**

Annotations to regulation of protein x-merization activity and children with evidence code ECO:0000304 (TAS)

Issue: Need manual review to possibly change to experimental evidence of 'regulation of protein complex assembly' or a child term; alternatively, can be removed.

**BHF-UCL** - 1
**ParkinsonsUK-UCL** - 1

**Comment by pfey03** [comment]
I looked at my 
tgrC1 GO:0090073 (positive regulation of protein homodimerization activity) has_regulation_target tgrB1

TgrB1 only homodimerizes upon interaction with TgrC1; tgrC1 homodimerizes spontanously but tgrB1 cannot homodimerize to form the kin recognition complex without tgrC1.

How would I otherwise annotate this?How in GO CAM? Thanks!

**Comment by pfey03** [comment]
OK, the second annotation I just inspected. The phosphpducin phlp1is necessary for Gbeta/gamma dimerization, or complex formation. This I could annotate to regulation of complex formation has_input X I suppose
Maybe I need to think for tgrB1 also about kin recognition complex and need to rethink these with extensions and their relations. Thanks!!

**Comment by EricaBakker** [comment]
Fixed the three assigned to TAIR. The spreadsheet was read only, so could not mark this in the document.

**Comment by tberardini** [comment]
I updated the TAIR lines on the spreadsheet. Can you please add write access for @ebakker2 in the GDrive for the future? Thanks.

**Comment by vanaukenk** [comment]
@pfey03 

For the TgrC1 example, would 'positive regulation of protein complex assembly' also work?

https://www.ncbi.nlm.nih.gov/pubmed/24490801

**Comment by pgaudet** [comment]
UniProt annotations fixed or disputed in P2GO. 

Thanks, Pascale

**Comment by pgaudet** [comment]
@pfey03 @vanaukenk Looks like TgrC1 and TgrB1 *are* a complex; I think the process annotation would be wrong in this case. Remove ?

**Comment by pgaudet** [comment]
@ebakker2 please email help-at-geneontology.org to give us the email you want to use to get access to the GO google drive. 

Thanks, Pascale

**Comment by pfey03** [comment]
@vanaukenk @pgaudet I was off for a couple of days. Now I looked at a few papers I annotated in the past  and this is what happens:
tgrC1 spontaneously binds in one cell. When it encounters a matching tgrB1 bearing cell it causes tgrB1 also to cluster. 
Furthermore,  it was found that tgrB1 has receptor activity and tgrC1 is the ligand; tgrB1 is phosphorylated upon interaction, which could be the mechanisms that it clusters when binding tgrC1. When matching clusters bind they form a GO:0098635 'protein complex involved in cell-cell adhesion'
So I deleted the annotation now but I wish I could have annotated something like: 
tgrC1 positively_regulates GO:0042802 'identical protein binding' has_input tgrB1 
Would that be acceptable in GO-CAM?
Thanks!!

**Comment by pgaudet** [comment]
I think this is valid according to the GO-CAM specs; however this is a very mechanistic description of what is happening, I would say this is more granular than what we would like to capture with GO-CAMs. 

@vanaukenk @thomaspd do you want to comment ? 

Thanks, Pascale

**Comment by thomaspd** [comment]
Many of the examples look like direct interactions, so in these cases we should make an MF annotation.

**Comment by RLovering** [comment]
BHF-UCL and HGNC-UCL done

**Comment by vanaukenk** [comment]
@pfey03 
I agree with @pgaudet that this sounds like a more mechanistic representation than what we have in mind for GO-CAM.
I wonder, though, if you could make a causal model using the MF of 'cell adhesion receptor activity' (although I think the definition here might need to be revised to include binding to another cell adhesion molecule)?
http://amigo.geneontology.org/amigo/term/GO:0004895

**Comment by pfey03** [comment]
@vanaukenk Thanks Kimberly. Maybe the def of GO:0004895 should be widened. 

But this tgrB/C complex is more than cel cell adhesion, the complex determines cell-cell recognition, the binding only happens in 'fitting' partner cells - kin recognition.

These genes are annotated to the processes cell-cell recognition and socially cooperative development, but functions to describe this cell-cell recognition are not available. Maybe a new term 'cell-cell recognition receptor activity' should be created, if determined that the cell-cell recognition is a function (I think so). 

Should exist in other systems I would think. Thanks!

**Comment by BarbaraCzub** [comment]
ARUK-UCL done

**Comment by BarbaraCzub** [comment]
ParkinsonsUK-UCL done
